spec/dummy/app/views/home_page/show.html.haml

```diff
@@ -1,21 +0,0 @@
-.container
-  .hero-unit
-    %h1 wow mom
-.container
-  .row
-    .span12
-      %h2 hello
-
-.container
-  .carousel.slide
-    .carousel-inner
-      .item.active
-        first item
-      .item
-        second item
-
-= container do
-  woah!
-
-= progress_bar do |pb|
-  = pb.bar(40)
```

Comment:
Since we're getting rid of the dummy app, we may want to move what components are available into the README